### DISCUSSION TITLE
Variant discovery predictions

### DISCUSSION THREAD

**Discussion by husamia** [jzhoulab/puffin#1] [Ideas]
For the human genome, we often have variants of unknown significance in non coding regions. It would be useful if we can get a list of regions from the TSS Analyzer to scan against our list to find out if there any variants which could potentially have an impact on the gene reguation through change in the TSS acvitation. I guess the question would be how to define it. It would be some threshold which can be determined with some training on known mutations. Can we say pick the higher than average affinity for causing TSS activation if they are changed for the entire genome?

**Comment by jzthree** [comment]
Currently this is better done with the code in this repository. We don't have the bandwidth to add this feature to the website in the near term, but we will consider that in the future.
